CogNative/models/Real-Time-Voice-Cloning/main.py

```diff
@@ -32,8 +32,12 @@
 vocoder.load_model(voc_weights)
 
 # DEFINE OUTPUT TEXT FOR VOICE CLONE
-text = input("Enter text for voice clone:\n")
-assert text[-1] == ".", f"{Fore.RED}ERROR: Punctuation missing."
+filename_txt = input(Fore.LIGHTGREEN_EX + "Enter name of input text file:\n")
+in_txt = Path(f"Clone_Tests/{filename_txt}.txt")
+assert os.path.exists(in_txt), f"{Fore.RED}ERROR: File not found."
+text_file = open(f"Clone_Tests/{filename_txt}.txt", "r")
+text_data = text_file.read()
+text_file.close()

```

Comment:
Do you want to replace this, to stay consistent. I am about to remove this in a PR, but just in case anyone is pulling this version, it shouldn't change.

Reply:
If you are changing the text input method than I'm fine with updating this to what you have.

Reply:
I'm fine with leaving this since it is going to change very soon anyway.